code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -592,6 +592,29 @@
 	cover.name = name
 	cover.desc = desc
 
+/obj/machinery/porta_turret/centcomm_shuttle

```

Comment:
That's weird, also ```check_anomalies``` is true by default, so no point = TRUE'ing it.

I guess it's fine to leave it as is (though I suspect this is revealing some kind of bug with turret code)